SWLOR.Game.Server/Feature/StatusEffectDefinition/BattleInsightStatusEffectDefinition.cs

```diff
@@ -55,8 +55,8 @@ private void BattleInsight2(StatusEffectBuilder builder)
                 .Replaces(StatusEffectType.BattleInsight1)
                 .TickAction((source, target, effectData) =>
                 {
-                    var effect = EffectAttackDecrease(8);
-                    effect = EffectLinkEffects(effect, EffectACDecrease(8));
+                    var effect = EffectAttackDecrease(25);
```

Comment:
Just to add on to what I said above - this actually caps at 20 in NWN. So you're getting 20 * 5 = 100 instead of 25 * 5 = 125 you may (or may not) be expecting.